src/report.py

```diff
@@ -0,0 +1,98 @@
+import csv
+from typing import List, Dict, Tuple
+
+
+def read_portfolio(filename: str) -> List[Dict]:
+    """
+    Reads a portfolio file and places each entry into a list.
+
+    :param filename: Name of the portfolio file
+    :type filename: str
+    :raises ValueError: Raises ValueError if line in the file is empty
+    :return: Returns list of portfolio entries
+    :rtype: List[Dict]
+    """
+    portfolio = []
+    with open(filename) as f:
+        rows = csv.reader(f)
+        headers = next(rows)
+        for row in rows:
+            if not row:
+                raise ValueError
+            newrow = (row[0], int(row[1]), float(row[2]))
+            portfolio.append(dict(zip(headers, newrow)))
+
+    return portfolio
+
+
+def read_prices(filename: str) -> Dict:
```

Comment:
You can update the static typing with the type of keys and values:
```suggestion
def read_prices(filename: str) -> Dict[str, float]:
```

Reply:
Same remark for the `filename` argument which should be `file_path`